test/parallel/test-buffer-indexof.js

```diff
@@ -81,43 +81,54 @@ assert.equal(Buffer.from('ff').indexOf(Buffer.from('f'), 1, 'ucs2'), -1);
 // test hex encoding
 assert.equal(
     Buffer.from(b.toString('hex'), 'hex')
-    .indexOf('64', 0, 'hex'), 3);
+    .indexOf('64', 0, 'hex'),
```

Comment:
Although I don't think it's really going to fly in this project. There's going to definitely be people that will prefer this:

```js
var a = foo.bar()
           .baz()
           .bip()
           .bap();
```

...instead of:

```js
var a = foo.bar()
  .baz()
  .bip()
  .bap();
```

It may be possible to right a rule smart enough to distinguish between that and other cases where it may make sense to indent-by-two rather than align, but I suspect there will be many edge cases. Indentation of `then()` in promise chains would seem to be a likely one too.